online-shop/src/Styles/index.css

```diff
@@ -0,0 +1,53 @@
+/*Add some CSS to each of the components to make them look nicer.*/
+
+/* Title */
+.title {
+    text-align: center;
+}
+
+/* Table itself */
+table {
```

Comment:
Evita folosirea tag-urilor de html in fisierele de css, ca o sa ai situatii cand un fisier iti suprascrie stilurile din alt fisier. Evita si id-urile pe cat posibil. Foloseste clase.
La fel si la urmatoarele tag-uri.

Reply:
Also, incearca sa pui stilurile pentru fiecare componenta in fisiere de css individuale, pe cat de mult posibil.